### PR TITLE
DDCE-1571: replaced ga with tracking consent

### PR DESCRIPTION
**Pull request by fraser-sprott**

### CODE REVIEW

File: test/controllers/WhatNextPageControllerSpec.scala [hunk 1]
```diff
@@ -413,16 +413,16 @@ class WhatNextPageControllerSpec extends PlaySpec with FakePBIKApplication with
   }
 
   "The calculateTaxYear method" should {
-    "return this year and next year if given true" in {
+    "return this tax year and next year if given true" in {
       val result = whatNextPageController.calculateTaxYear(true)
-      assert(result._1 == LocalDate.now().getYear)
-      assert(result._2 == (LocalDate.now().getYear + 1))
+      assert(result._1 == 2020)
```
Comment (abhijeetardale) [inline test/controllers/WhatNextPageControllerSpec.scala:418]: any reason for adding constant values, earlier was more danyamic date

File: test/controllers/WhatNextPageControllerSpec.scala [hunk 2]
```diff
@@ -413,16 +413,16 @@ class WhatNextPageControllerSpec extends PlaySpec with FakePBIKApplication with
...
+      assert(result._2 == 2021)
     }
 
     "return next year and the year after if given false" in {
       val result = whatNextPageController.calculateTaxYear(false)
-      assert(result._1 == (LocalDate.now().getYear + 1))
-      assert(result._2 == (LocalDate.now().getYear + 2))
+      assert(result._1 == 2021)
```
Comment (abhijeetardale) [inline test/controllers/WhatNextPageControllerSpec.scala:424]: same as above

**Review by abhijeetardale — CHANGES_REQUESTED** [review]

File: test/controllers/WhatNextPageControllerSpec.scala (hunk 1 above)
Comment (fraser-sprott) [reply test/controllers/WhatNextPageControllerSpec.scala:418]: the dynamic one wasn't working for the case when the tax year moves into the new calander year, e.g april 2020 to april 2021. If we were to change the test validattion to include this we are essentially creating the method we are testing, agree not ideal but awkward scenario

Comment (abhijeetardale) [reply test/controllers/WhatNextPageControllerSpec.scala:418]: Its possible to resolve tax year issue with TaxYear library

**Review by abhijeetardale — APPROVED** [review]

**Pull request merged**